### PR TITLE
House-Price-Prediction

### PR DESCRIPTION
**Pull request by Vanshikagoel0012**
## Issue Title :

- Info about the related issue (Aim of the project)

- [x] SWOC Participant 
- [ ] Contributor

Closes: #issue number that will be closed through this PR

### Describe the add-ons or changes you've made

Give a clear description of what have you added or modifications made

## Type of change

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested?

Describe how it has been tested
Describe how have you verified the changes made

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [x] I have added things that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Review by abhisheks008 — CHANGES_REQUESTED** [review]
1. Delete the unnecessary files that you have attached in the main branch.
2. Follow the proper structure of the README.md file.
3. Use 4-5 models and make a comparison to achieve the best fitted model.
4. For reference you can take a look at the others' projects.

**Comment by Vanshikagoel0012** [comment]
Sir, I have  followed the proper process of it and also readme is attached and extra file are not visible to time kindly point out that

File: readme.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Linear-Regression-Model-for-House-Price-Prediction.
```
Comment (abhisheks008) [inline readme.txt:1]: delete these kind of files which are attached directly in main repo not in your project folder

**Review by abhisheks008 — CHANGES_REQUESTED** [review]

**Comment by abhisheks008** [comment]
And you have not followed the format of README.md file. Please follow the proper format. Use more algorithms (4-5) and justify the best fitted model.

**Comment by abhisheks008** [comment]
https://github.com/abhisheks008/ML-Crate/blob/main/.github/readme_template.md
Follow this.

**Pull request closed without merging**